alerts/actions/triage_bot.py

```diff
@@ -0,0 +1,746 @@
+    null = {"documentsource": {"details": {"object": None}}}
+
+    _source = alert.get("_source", {})
+    _events = _source.get("events", [null])
+
+    email = _events[0]["documentsource"]["details"]["object"]
+
+    if email is None or email == "":
+        return None
+
+    summary = (
+        "DUO bypass codes belonging to your account have been used to "
+        "authenticate.  This should only happen in the case of the loss of other "
+        "less secret credentials."
+    )
+
+    return AlertTriageRequest(
+        alert["_id"],
+        AlertLabel.DUO_BYPASS_CODES_USED,
+        summary,
+        email,
+        Confidence.HIGHEST,
+    )
+
+
+def _make_ssh_access_releng(
+    alert: Alert, tkn: Token
+) -> types.Optional[AlertTriageRequest]:
+    null = {"documentsource": {"details": {"hostname": None}}}
+
+    _source = alert.get("_source", {})
+    _events = _source.get("events", [null])
+
+    user = _source.get("summary", "").split(" ")[-1]
+    host = _events[0]["documentsource"]["details"]["hostname"]
+
+    if user == "" or host is None or host == "":
+        return None
+
+    confidence = Confidence.HIGH
+    profile = primary_username(PERSON_API_BASE, tkn, user)
+
+    if profile is None:
+        profile = User(
+            created=datetime.now(),
+            first_name="",
+            last_name="",
+            alternative_name="",
+            primary_email="{}@mozilla.com".format(user),
+            mozilla_ldap_primary_email="",
+        )
+
+        confidence = Confidence.LOW
+
+    summary = (
+        "An SSH session was established to host {} by your user " "account."
+    ).format(host)
+
+    return AlertTriageRequest(
+        alert["_id"],
+        AlertLabel.SSH_ACCESS_SIGN_RELENG,
+        summary,
+        profile.primary_email,
+        confidence,
+    )
+
+
+def _retrieve_duplicate_chain(
+    api: RESTConfig, label: AlertLabel, email: Email
+) -> types.Optional[DuplicateChain]:
+    url = "{}/alerttriagechain".format(api.url)
+
+    payload = {
+        "alert": label.value,
+        "user": email,
+    }
+
+    jwt_auth = None
+
+    if api.token is not None:
+        jwt_auth = JWTAuth(api.token)
+        jwt_auth.set_header_format("Bearer %s")
+
+    try:
+        resp = requests.get(url, params=payload, auth=jwt_auth)
+        resp_data = resp.json()
+    except json.JSONDecodeError as ex:
+        raise APIError("Did not receive JSON response: {}".format(ex))
+    except requests.exceptions.RequestException as ex:
+        raise APIError("Failed to make request: {}".format(ex))
+
+    error = resp_data.get("error")
+
+    if error is not None:
+        if resp.status_code != 200:
+            raise APIError(error)
+
+        return None  # No duplicate chain found
+
+    ids = resp_data.get("identifiers", [])
+
+    if len(ids) == 0:
+        return None
+
+    try:
+        created = datetime.strptime(resp_data["created"], DUP_CHAIN_DATE_FMT)
```

Comment:
Instead of needing to create DUP_CHAIN_DATE_FMT, we should use https://github.com/mozilla/MozDef/blob/master/mozdef_util/mozdef_util/utilities/toUTC.py#L10

Reply:
Would this change also necessitate moving from `datetime.utcnow()` to `datetime.now()` or is that fine as is?

Reply:
It shouldn't be a problem, because while `datetime.now()` doesn't include the timezone, we automatically assume it's UTC in toUTC if a timezone isn't specified.